sage-ios/SAGE/SAGE/LoginView.swift

```diff
@@ -0,0 +1,236 @@
+//
+//  LoginView.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 10/7/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+
+import UIKit
+
+class LoginView: UIView {
+    
+    var loginEmailField: UITextField?
+    var loginPasswordField: UITextField?
+    var signUpLink: UIButton?
+    var sageLabel: UILabel?
+    var containerView: UIView?
+    var firstDivider: UIView?
+    var secondDivider: UIView?
+    var loginButton: UIButton?
+    var loginButtonText: UILabel?
+    
+    var movedUp: Bool = false
+    
+    required override init(frame: CGRect) {
+        let screenRect = UIScreen.mainScreen().bounds;
+        let screenWidth = screenRect.size.width;
+        let screenHeight = screenRect.size.height;
+        let newFrame = CGRectMake(0, 0, screenWidth, screenHeight)
+        super.init(frame: newFrame)
```

Comment:
so after init, the viewcontroller will automatically resize its view property so that it fills the screen, and when that happens, the view's layoutSubviews will get called, so you actually dont need to worry about what frame is passed in at all!